src/SFA.DAS.QnA.Application/Commands/ResetPageAnswers/ResetPageAnswersHandler.cs

```diff
@@ -19,6 +17,7 @@ public ResetPageAnswersHandler(QnaDataContext dataContext, INotRequiredProcessor
 
         public async Task<HandlerResponse<ResetPageAnswersResponse>> Handle(ResetPageAnswersRequest request, CancellationToken cancellationToken)
         {
+            var application = await _dataContext.Applications.SingleOrDefaultAsync(app => app.Id == request.ApplicationId, cancellationToken);
```

Comment:
I'd move this back to being below the validationErrorResponse so you're not wasting a database call if validation fails

Reply:
Done